modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/SearchBar.js

```diff
@@ -35,23 +36,15 @@ export default function SearchBar({
 	scopeParameterStringEverything,
 	searchURL,
 	selectedEverythingSearchScope = false,
-	suggestionsContributorConfiguration = '{}',
+	suggestionsContributorConfiguration = '[]',
 	suggestionsDisplayThreshold = '2',
 	suggestionsURL = '/o/portal-search-rest/v1.0/suggestions',
 }) {
-	const fetchURL = new URL(
-		`${Liferay.ThemeDisplay.getPathContext()}${suggestionsURL}`,
-		Liferay.ThemeDisplay.getPortalURL()
-	);
-
 	const [active, setActive] = useState(false);
 	const [autocompleteSearchValue, setAutocompleteSearchValue] = useState('');
 	const [inputValue, setInputValue] = useState(keywords);
-	const [networkState, setNetworkState] = useState(() => ({
-		error: false,
-		loading: false,
-		networkStatus: 4,
-	}));
+	const [loading, setLoading] = useState(false);
+	const [resourceItems, setResourceItems] = useState([]);
```

Comment:
Let's make `resourceItems` more specific. Perhaps renaming it to `suggestions`?

Reply:
Oh I was wondering what to name this, but I think there is already a `suggestions` object within that data?

Reply:
Maybe `suggestionsResponseData`?

Reply:
Or `suggestionsResponseItems`

Reply:
Thanks! I'll go with `suggestionsResponseItems` if that's all right!